charts/kong/README.md

```diff
@@ -288,6 +293,124 @@ scenario and several settings that are useful when using multiple controllers:
+Kong Enterprise 2.1.0. These deployments split Kong nodes into control plane
+(CP) nodes, which provide the admin API and interact with the database, and
+data plane (DP) nodes, which provide the proxy and receive configuration from
+control plane nodes.
+
+The chart does not support deploying control and data plane nodes in a single
+release. Users should [create separate releases for each node
+type](#separate-admin-and-proxy-nodes), i.e. use separate control and data
+plane values.yamls that are installed separately.
+
+* If `.Values.env.role="control_plane"`, the release will create a Service
+  named `RELEASE-NAME-kong-cluster`.
+* Cluster certificates are not yet generated automatically. You must create a
+  certificate/key pair from a local instance, create a Secret containing them,
+  add the Secret to `.Values.secretVolumes`, and add
+  `cluster_cert`/`cluster_cert_key` values indicating its location under
+  `/etc/secrets` to `env`.
+* Note that data plane nodes do not default to using the default port (8005)
+  for `cluster_control_plane`. You should use `RELEASE-NAME-kong-cluster:8005`
+  as the `cluster_control_plane` value if using the default port.
+
+#### Certificates
+
+Hybrid mode uses TLS to secure the CP/DP node communication channel, and
+requires certificates for it. You can generate these either using `kong hybrid
+gen_cert` on a local Kong installation or using OpenSSL:
+
+```
+openssl req -new -x509 -nodes -newkey ec:<(openssl ecparam -name secp384r1) \
+  -keyout /tmp/cluster.key -out /tmp/cluster.crt \
+  -days 1095 -subj "/CN=kong_clustering"
+```
+
+You must then place these certificates in a Secret:
+
+```
+kubectl create secret tls kong-cluster-cert --cert=/tmp/cluster.crt --key=/tmp/cluster.key
+```
+
+#### Control plane node configuration
+
+You must configure the control plane nodes to mount the certificate secret on
+the container filesystem is serve it from the cluster listen. In values.yaml:
+
+```yaml
+secretVolumes:
+- kong-cluster-cert
+```
+
+```yaml
+env:
+  role: control_plane
+  cluster_cert: /etc/secrets/kong-cluster-cert/tls.crt
+  cluster_cert_key: /etc/secrets/kong-cluster-cert/tls.key
+```
+
+Furthermore, you must enable the cluster listen and Kubernetes Service, and
+should typically disable the proxy:
+
+```
+cluster:
+  enabled: false
+  tls:
+    enabled: true
+    servicePort: 8005
+    containerPort: 8005
+
+proxy:
+  enabled: false
+```
+
+Note that this does not enable an Ingress for the cluster listen or configure
+the cert on the Ingress. This may be necessary if you run CP and DP nodes in
+separate clusters depending on how you expose the clusters' networks to one
+another.
+
+#### Data plane node configuration
+
+Data plane configuration also requires the certificate and `role`
+configuration, and the database should always be set to `off`. You must also
+trust the cluster certificate and indicate what hostname/port it should use to
+find control plane nodes.
```

Comment:
misleading pronouns:

> **You** must also trust the client certificate

> indicate what hostname/port **it** should use

suggestion: replace those pronouns with "Kong"

Reply:
the former case not updated - mistake?

I think that _"You"_ is not appropriate here because it's probably the server (and not the human reading this doc which is pointed at by "you") who needs to trust that client certificate.

Reply:
Usually if I only update something in part and resolve it means I only agreed with part--I won't always call out stuff I didn't do explicitly if I didn't think it was a big deal. This is keeping with https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md#personal-pronouns

This is more "you must do something" as a call to action; the longer implied form is "you must configure the data plane node to trust the certificate". The configuration example after shows exactly what to do in the actual config, so I'm less concerned with the exact specificity and absolute correctness of the presentation before.